resources/js/src/import.ts

```diff
@@ -1,6 +1,6 @@
 import $ from 'jquery';
 import { AJAX } from './modules/ajax.ts';
-import { Functions } from './modules/functions.ts';
+import { formatBytes } from './modules/functions.ts';
```

Comment:
```suggestion
import * as Functions from './modules/functions.ts';
```
That would also help stack traces be readable?
For the Functions is undefined bug

Reply:
> That would also help stack traces be readable?

That will make the import.ts import all exported stuff from functions.ts instead of only the formatBytes function. I think the stack trace issue has more to do with the source maps not being used by the report tool.

> For the Functions is undefined bug

Are you talking about a specific bug? `Functions` doesn't exist anymore with this PR.

Reply:
> I think the stack trace issue has more to do with the source maps not being used by the report tool.

Definitly